### ISSUE TITLE
Parse as string

### ISSUE THREAD

**Issue by scinfu** [woahdae/simple_xlsx_reader#35]
In my sheet I have some value than is transformed in date or number. 
How can I force all to be parsed as a string?

**Comment by woahdae** [comment]
I don't believe this functionality exists, but probably wouldn't be hard to add, since you'd just be bypassing the code that does casting.

**Comment by laptopmutia** [comment]
@woahdae so to avoid the SimpleXlsxReader appending .0 to my ids/foreign_key column

I need to cast it manually to integer then to string right?

is there any other way?

what I'm currently doing is this

```ruby
raw_parent_id = row["parent_id"]
parent_id = if raw_parent_id.is_a?(Numeric)
  raw_parent_id.to_i.to_s
else
  raw_parent_id
end
```
